extensions/smallrye-jwt/deployment/src/main/java/io/quarkus/smallrye/jwt/deployment/SmallRyeJwtProcessor.java

```diff
@@ -66,6 +74,17 @@ class SmallRyeJwtProcessor {
 
     private static final String MP_JWT_VERIFY_KEY_LOCATION = "mp.jwt.verify.publickey.location";
     private static final String MP_JWT_DECRYPT_KEY_LOCATION = "mp.jwt.decrypt.key.location";
+    private static final String MP_JWT_VERIFY_PUBLIC_KEY = "mp.jwt.verify.publickey";
+
+    private static final String SMALLRYE_JWT_SIGN_KEY_LOCATION = "smallrye.jwt.sign.key.location";
+    private static final String SMALLRYE_JWT_SIGN_KEY = "smallrye.jwt.sign.key";
+
+    private static final Set<String> JWT_KEY_PROPERTIES = Set.of(
```

Comment:
```suggestion
    private static final Set<String> JWT_SIGN_KEY_PROPERTIES = Set.of(
```

Propose to group all keys related to signing/verifying only, we can deal with the encryption/decryption later